fuel_test/deploy_env.sh

```diff
@@ -0,0 +1,287 @@
+#!/bin/bash
+
+set -x
+
+function wait_for_fm {
+	# Wait for fuel master booting and return its IP address
+	local xs_host="$1"
+	local fm_name="$2"
+	local retry_count=$3
+	local retry_interval=$4
+
+	local fm_ip
+	local counter=0
+	while [ $counter -lt $retry_count ]; do
+		fm_ip=$(ssh -qo StrictHostKeyChecking=no root@$xs_host \
```

Comment:
Simplify the SSH command and run the rest in this script; e.g.
fm_networks=$(ssh ... xe vm-list uuid=<uuid> params=networks --minimal)
fm_ip=$(echo $fm_networks | egrep ...)